h5c/vic/src/vic-webapp/e2e/app.po.ts

```diff
@@ -141,13 +143,13 @@ export class VicWebappPage {
   }
 
   deleteVch(vch) {
-    browser.ignoreSynchronization = true;
     this.waitForElementToBePresent(this.actionBar + vch);
     this.clickByCSS(this.actionBar + vch);
     this.clickByCSS('button.' + vch);
     browser.switchTo().defaultContent();
     this.waitForElementToBePresent(this.iframeModal);
     this.switchFrame(this.iframeModal);
+    this.waitForElementToBePresent(this.labelDeleteVolumes);
```

Comment:
In the future, it'd probably be better to put unrelated changes into their own PR.

Reply:
I think it makes sense to update E2E test case logics at the same time especially when you updated a component or service that might affect the way the UI is behaving. This PR is going to cause a spinner to spin until clusters and standalone hosts are loaded, which was not pretty much the case with the status quo, which was already in a broken state but somehow failed silently.